lm_eval/evaluator.py

```diff
@@ -20,17 +22,24 @@
     eval_logger,
 )
 
+if TYPE_CHECKING:
+    from lm_eval.api.model import LM
+    from lm_eval.tasks import Task
+
 
 @positional_deprecated
 def simple_evaluate(
     model,
     model_args=None,
-    tasks=None,
+    model_arg_dict=None,
```

Comment:
could we have `model_args` simply be of either type `str` or `dict` and change behavior based on that?

Reply:
Absolutely, could benefit from some polymorphism, at a second glance, that makes a lot of sense, I'll make the change